trestle/transforms/utils/osco_helper.py

```diff
@@ -61,36 +61,32 @@
 class RuleUse():
     """Represents one rule of OSCO data."""
 
-    def __init__(
-        self,
-        id_: str,
-        target: str,
-        target_type: str,
-        benchmark_href: str,
-        benchmark_id: str,
-        scanner_name: str,
-        scanner_version: str,
-        idref: str,
-        version: str,
-        time: str,
-        result: str,
-        severity: str,
-        weight: str
-    ) -> None:
+    def __init__(self, **args) -> None:
         """Initialize given specified args."""
-        self.id_ = id_
-        self.target = target
-        self.target_type = target_type
-        self.benchmark_href = benchmark_href
-        self.benchmark_id = benchmark_id
-        self.scanner_name = scanner_name
-        self.scanner_version = scanner_version
-        self.idref = idref
-        self.version = version
-        self.time = time
-        self.result = result
-        self.severity = severity
-        self.weight = weight
+        self.id_ = args['id_']
+        self.target = args['target']
+        self.target_type = args['target_type']
+        self.host_name = args['host_name']
+        self.benchmark_href = args['benchmark_href']
+        self.benchmark_id = args['benchmark_id']
+        self.scanner_name = args['scanner_name']
+        self.scanner_version = args['scanner_version']
+        self.idref = args['idref']
+        self.version = args['version']
+        self.time = args['time']
+        self.result = args['result']
+        self.severity = args['severity']
+        self.weight = args['weight']
+
+    @property
+    def inventory_key(self):
+        """Derive inventory key."""
+        if self.host_name is None:
+            # OpenScap 1.3.3
+            return self.target + ':' + self.target_type
+        else:
```

Comment:
You don't need the else here as you have returned.

Reply:
Fixed.